docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -0,0 +1,83 @@
+---
+---
+= Clustered Cache
+ifndef::sourcedir[]
+include::common.adoc[]
+endif::sourcedir[]
+
+ifdef::notBuildingForSite[]
+include::menu.adoc[]
+endif::notBuildingForSite[]
+
+== Introduction
+
+To enable Clustering with Terracotta, firstly you will have to start the http://terracotta.org[Terracotta server] configured with clustered storage.
+And then need a cache manager with clustering capabilities so that the caches can consume the clustered storage.
+
+== Starting Terracotta server
+
+Assuming that you have the clustered EhCache kit available locally, start with extracting the *ehcache-clustered* kit.
+Change to your extracted directory and then execute the *start-tc-server* script as below to start the Terracotta server with default configuration:
+
+On Windows: +
+Execute the *terracotta-5.x.y-SNAPSHOT/server/bin/start-tc-server.bat* script.
+
+On Unix/Mac:
+[source,bash]
+----
+cd terracotta-5.x.y-SNAPSHOT/server/bin
+./start-tc-server.sh
+----
+
+NOTE: You will need to have `JAVA_HOME` set to JDK8 while starting the Terracotta server.
+
+Check for the below `INFO` log to confirm if the server started successfully,
+`Terracotta Server instance has started up as ACTIVE node on 0:0:0:0:0:0:0:0:9510 successfully, and is now ready for work.`
```

Comment:
That looks like a nice IPv6 log output.
Any change we can have something more friendly that says localhost for example?
If that requires config tweaks, then forget about it.